src/webui/src/components/trial-detail/Para.tsx

```diff
@@ -238,15 +238,19 @@ class Para extends React.Component<ParaProps, ParaState> {
                         },
                         axisLabel: {
                             formatter: function (value: string) {
-                                const length = value.length;
-                                if (length > 16) {
-                                    const temp = value.split('');
-                                    for (let m = 16; m < temp.length; m += 17) {
-                                        temp[m] += '\n';
+                                if (value !== undefined) {
```

Comment:
The type annotation says it can't be `undefined`?

Reply:
Ok, will add it.